aidb/engine/approx_aggregate_engine.py

```diff
@@ -30,9 +30,6 @@ async def execute_aggregate_query(self, query: Query):
       if len(engine_required) == 0:
         blob_key_filtering_predicates.append(filtering_predicate)
     blob_key_filtering_predicates_str = self._get_where_str(blob_key_filtering_predicates)
-    column_to_root_column = self._config.columns_to_root_column
-    for k, v in column_to_root_column.items():
-      blob_key_filtering_predicates_str = blob_key_filtering_predicates_str.replace(k, v)

```

Comment:
https://github.com/ddkang/aidb/blob/0bff83f87e1008abcb98cd2138bf7da54a5bc3c3/aidb/query/query.py#L231-L249
https://github.com/ddkang/aidb/blob/0bff83f87e1008abcb98cd2138bf7da54a5bc3c3/aidb/query/query.py#L281-L286
This has been moved into query.py. Because we don't want to change those columns' name in subquery